deploy/maistra-operator.yaml

```diff
@@ -6975,7 +6980,7 @@ spec:
         olm.relatedImage.v1_1.proxyv2: docker.io/maistra/proxyv2-ubi8:1.1.8
         olm.relatedImage.v1_1.sidecar-injector: docker.io/maistra/sidecar-injector-ubi8:1.1.8
 
-        olm.relatedImage.v2_0.3scale-istio-adapter: quay.io/3scale/3scale-istio-adapter:v2.0.0
+        olm.relatedImage.v2_0.3scale-istio-adapter: quay.io/3scale/3scale-istio-adapter:v1.0.0
```

Comment:
I think 3scale is publishing 2.0.0 for 2.0

Reply:
and we don't usually test 3scal either. That's usually done by the 3scale team.

Reply:
This change came in when I ran `make generate-manifests`, so it was already in the codebase somewhere. I've now removed it from this PR.

Reply:
I have the same change in my PR. please just leave it in- if we want to make it 2.0.0 we'll have to change the template that sets it to 1.0.0 in a separate PR. I will also make sure we're checking this in `gen-check` in the future

Reply:
Ok. The threescale guys made the change to upgrade it to 2.0. Perhaps they made it in the wrong location? Approving.